src/app/services/category.service.js

```diff
@@ -97,6 +98,12 @@ class CategoryService {
     }
 
     formatAutocompleteQuery(resourceId, filters, targetField) {
+
+        if ( ! targetField || targetField === '') {
+            console.log('targetField is missing');
```

Comment:
should we log all the time?

Reply:
I am going to amend that part as logging only if a flag is on. The flag is going to come from the new service I am mentioning below.